.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -0,0 +1,145 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  ASDF_MAGE_VERSION: 1.14.0
+
+  IMAGE_UBUNTU_2404_X86_64: "platform-ingest-elastic-agent-ubuntu-2404-1749258065"
+  IMAGE_UBUNTU_X86_64_FIPS: "platform-ingest-elastic-agent-ubuntu-2204-fips-1748955449"
+  IMAGE_UBUNTU_ARM64_FIPS: "platform-ingest-elastic-agent-ubuntu-2204-fips-aarch64-1748955449"
```

Comment:
Just a question, shall we use `-fips` after `-aarch64` and follow the below naming convention?

- `platform-ingest-elastic-agent-<os-name>-<os-version>(-fips)?(-<arch>)?-<timestamp>`

where:
- `<os-name>` is `ubuntu`, `windows`, `debian`, ...
- `<os-version>` is `2204`, `11`, `10`, `2025`, ...
- `<arch>` is empty for `amd64` or `aarch64` ...
- `<timestamp>` is current timemilliseconds.

I somehow think `-fips` should be at the end similarly done with the name of the Env variable `IMAGE_UBUNTU_ARM64_FIPS`. What do you think?

Reply:
@pchila, I know we recently had discussions for renaming `elastic-agent-fips-cloud -> elastic-agent-cloud-fips`; do we have an option for VM image names?